private/bufpkg/bufmodule/module_set.go

```diff
@@ -190,6 +190,40 @@ func ModuleSetToDAG(moduleSet ModuleSet) (*dag.Graph[string], error) {
 	return graph, nil
 }
 
+// ModuleToSelfContainedModuleReadBucketWithOnlyProtoFiles converts the Module to a
```

Comment:
Why did this move?

Reply:
Oh, I think I moved this by accident while traversing some of the helpers. Moved it back.